docs/en_US/Tutorial/HowToLaunchFromPython.rst

```diff
@@ -90,6 +90,213 @@ Below is an example for this new launching approach. You can also find this code
+
+    search_space = {
+        "C": {"_type":"quniform","_value":[0.1, 1, 0.1]},
+        "kernel": {"_type":"choice","_value":["linear", "rbf", "poly", "sigmoid"]},
+        "degree": {"_type":"choice","_value":[1, 2, 3, 4]},
+        "gamma": {"_type":"quniform","_value":[0.01, 0.1, 0.01]},
+        "coef0": {"_type":"quniform","_value":[0.01, 0.1, 0.01]}
+    }
+
+3. Configure Experiment
+"""""""""""""""""""""""
+
+.. code:: ipython3
+
+    from nni.experiment import Experiment
+    experiment = Experiment(tuner, 'local')
+    experiment.config.experiment_name = 'test'
+    experiment.config.trial_concurrency = 2
+    experiment.config.max_trial_number = 5
+    experiment.config.search_space = search_space
+    experiment.config.trial_command = 'python3 main.py'
+    experiment.config.trial_code_directory = './'
+
+4. Start Experiment
+"""""""""""""""""""
+
+.. code:: ipython3
+
+    experiment.start(8081)
+
+.. parsed-literal::
+
+    [2021-02-22 12:27:11] Creating experiment, Experiment ID: bj025qo4
+    [2021-02-22 12:27:11] Connecting IPC pipe...
+    [2021-02-22 12:27:15] Statring web server...
+    [2021-02-22 12:27:16] Setting up...
+    [2021-02-22 12:27:16] Dispatcher started
+    [2021-02-22 12:27:16] Web UI URLs: http://127.0.0.1:8081 http://10.0.1.5:8081 http://172.17.0.1:8081
+
+5. Experiment View & Control
+""""""""""""""""""""""""""""
+
+.. code:: ipython3
+
+    experiment.get_status()
+
+.. parsed-literal::
+
+    'RUNNING'
+
+.. code:: ipython3
+
+    experiment.export_data()
+
+.. parsed-literal::
+
+    [TrialResult(parameter={'coef0': 0.01, 'gamma': 0.01, 'degree': 1, 'kernel': 'linear', 'C': 0.1}, value=0.9866666666666667, trialJobId='B55mT'),
+     TrialResult(parameter={'coef0': 0.02, 'gamma': 0.01, 'degree': 1, 'kernel': 'linear', 'C': 0.1}, value=0.9866666666666667, trialJobId='QkhD0')]
+
+.. code:: ipython3
+
+    experiment.get_job_metrics()
+
+.. parsed-literal::
+
+    {'B55mT': [TrialMetricData(timestamp=1613996853005, trialJobId='B55mT', parameterId='0', type='FINAL', sequence=0, data=0.9866666666666667)],
+     'QkhD0': [TrialMetricData(timestamp=1613996853843, trialJobId='QkhD0', parameterId='1', type='FINAL', sequence=0, data=0.9866666666666667)]}
+
+6. Stop Experiment
+""""""""""""""""""
+
+.. code:: ipython3
+
+    experiment.stop()
+
+.. parsed-literal::
+
+    [2021-02-22 12:28:16] Stopping experiment, please wait...
+    [2021-02-22 12:28:16] Dispatcher exiting...
+    [2021-02-22 12:28:17] Experiment stopped
+    [2021-02-22 12:28:19] Dispatcher terminiated
+
+Connect and Manage an Exist Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Initialize Experiment without Tuner
+""""""""""""""""""""""""""""""""""""""
+
+.. code:: ipython3
+
+    from nni.experiment import Experiment
+    experiment = Experiment()
+
+.. parsed-literal::
+
+    [2021-02-22 12:29:01] Tuner not set, wait for connect...
+
+2. Connect Experiment
+"""""""""""""""""""""
+
+.. code:: ipython3
+
+    experiment.connect(8080)
```

Comment:
which one is correct? `Experiment.connect()`?

Reply:
for now, `experiment.connect()`.